recipe/meta.yaml

```diff
@@ -1,28 +1,26 @@
 {% set name = "cairocffi" %}
 {% set version = "0.9.0" %}
-{% set sha256 = "4600529463a8ece4bbdaf82e14a78769eaf143a8559e1b5469004f3fb8e7ff4d" %}
 
 package:
   name: cairocffi
   version: "{{ version }}"
 
 source:
   fn: {{ name }}-{{ version }}.tar.gz
-  url: https://github.com/Kozea/{{ name }}/archive/v{{ version }}.tar.gz
-  sha256: {{ sha256 }}
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 15386c3a9e08823d6826c4491eaccc7b7254b1dc587a3b9ce60c350c3f990337
   patches:
-    - dlopen-patch.linux.txt  # [linux]
-    - dlopen-patch.osx.txt  # [osx]
+    - dlopen-patch.diff
 
 build:
-  number: 0
-  skip: True  # [win]
-  script: python setup.py install --single-version-externally-managed --record record.txt
+  noarch: python
```

Comment:
I'm not familiar with this package, can it be `noarch`?
If so, why it was skipping Windows before?

Reply:
hmmm, good question! I think it can, since these are only python bindings to  the cffi library (which is of course platform-dependent). Searching for the library files is a little bumpy across archs but that is taken care of in the patch (as you suggested https://github.com/conda-forge/conda-smithy/issues/1050#issuecomment-474431785).

Reply:
> If so, why it was skipping Windows before?

I have absolutely no idea, sorry!

Reply:
OK. Let's try it and, if we find any problems, we can pull the package.